persistence-modules/spring-data-jpa-4/src/main/resources/application.properties

```diff
@@ -2,4 +2,12 @@ spring.jpa.show-sql=true
 #MySql
 #spring.datasource.url=jdbc:mysql://localhost:3306/baeldung
 #spring.datasource.username=baeldung
-#spring.datasource.password=baeldung
\ No newline at end of file
+#spring.datasource.password=baeldung
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.jpa.database-platform=org.hibernate.dialect.MySQL57Dialect
+
+#JPA-Schema-Generation
+#spring.jpa.properties.javax.persistence.schema-generation.scripts.action=create
+#spring.jpa.properties.javax.persistence.schema-generation.scripts.create-target=create.sql
+#spring.jpa.properties.javax.persistence.schema-generation.scripts.create-source=metadata
+#spring.jpa.properties.hibernate.format_sql=true
```

Comment:
These are all commented out - should you leave a comment showing what the settings are and when to uncomment them?

Reply:
Yes, good point. Done.